sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_train_labeled_model_async.py

```diff
@@ -34,15 +34,17 @@ class TrainLabeledModelSampleAsync(object):
     container_sas_url = os.environ["CONTAINER_SAS_URL"]
 
     async def train_labeled_model(self):
+        # [START create_form_training_client_async]
         from azure.ai.formrecognizer.aio import FormTrainingClient
         from azure.core.credentials import AzureKeyCredential
 
-        async with FormTrainingClient(
+        form_training_client = FormTrainingClient(
             endpoint=self.endpoint, credential=AzureKeyCredential(self.key)
-        ) as form_training_client:
+        )
+        # [END create_form_training_client_async]
+        async with form_training_client:
 
             model = await form_training_client.training(self.container_sas_url, use_labels=True)
```

Comment:
Is it worth adding some comments here, or links to service docs, to explain how label files are created and that they should be stored in the blob container with the other form files?

Reply:
Oh I see you have this above.